osx/bitbar.1m.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+out=$(python /Users/nathanevans/Documents/main.py)
```

Comment:
nit x2: full path
nit: according to the bitbar docs, you should be able to name your `main.py` => `bitbar.1m.py` (or preferably `status.1m.py`) and not need a shell script at all